src/api/burn/WixToolset.Mba.Core/IBootstrapperApplication.cs

```diff
@@ -2150,4 +2154,21 @@ public enum INSTALLUILEVEL
         /// </summary>
         UacOnly = 0x200,
     }
+
+    /// <summary>
+    /// The available types of hashes.
+    /// </summary>
+    public enum BOOTSTRAPPER_UPDATE_HASH_TYPE
```

Comment:
This enum already exists as `UpdateHashType` in IBootstrapperEngine.cs which already existed in wix3. It looks like I missed updating that enum when changing to SHA512.